consensus/src/dag/reliable_broadcast.rs

```diff
@@ -70,29 +78,23 @@ impl ReliableBroadcast {
                     )
                 }
             };
+            let network_message = message.into_network_message();
```

Comment:
Should we use DAGNetworkMessage throughout and convert between ConsensusMsg in dag_handler and dag_network_sender?